.github/workflows/test.yml

```diff
@@ -7,23 +7,32 @@ on:
   workflow_dispatch:
 
 jobs:
-  short:
+  devrequirements:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
-      - run: make test
+      - run: make install-buf
```

Comment:
Is the `devrequirements` job necessary any more now that the `verify-generated-code` job also runs `make install-buf`?

Reply:
Maybe not, but I sort of like the explicitness of it. Will fence sit on it for a bit.